### ISSUE TITLE
Which prices shall we use?

### ISSUE THREAD

**Issue by rmakara** [DivanteLtd/shopware-pwa#263]

**Comment by rmakara** [comment]
List prices added to API.

**Comment by patzick** [comment]
related: #357 
* https://github.com/DivanteLtd/shopware-pwa/pull/357/files#diff-dfd8fa17a92d387f380e8495ed8c5187R93
* https://github.com/DivanteLtd/shopware-pwa/pull/357/files#diff-dfd8fa17a92d387f380e8495ed8c5187R99

**Comment by elkmod** [comment]
Hey, hopefully this clears it up a bit:

```
{
  "calculatedListingPrice": { // Listing price = Original price
    "from": {
      "unitPrice": 123,
      "quantity": 1,
      "totalPrice": 123,
      "calculatedTaxes": [
        {
          "tax": 8.05,
          "taxRate": 7,
          "price": 123,
          "extensions": []
        }
      ],
      "taxRules": [
        {
          "taxRate": 7,
          "percentage": 100,
          "extensions": []
        }
      ],
      "referencePrice": null,
      "listPrice": null,
      "extensions": []
    },
    "to": {
      "unitPrice": 123,
      "quantity": 1,
      "totalPrice": 123,
      "calculatedTaxes": [
        {
          "tax": 8.05,
          "taxRate": 7,
          "price": 123,
          "extensions": []
        }
      ],
      "taxRules": [
        {
          "taxRate": 7,
          "percentage": 100,
          "extensions": []
        }
      ],
      "referencePrice": null,
      "listPrice": null,
      "extensions": []
    },
    "extensions": []
  },
  "calculatedPrices": [], // Used for multi tier pricing
  "calculatedPrice": { // Used single tier pricing
    "unitPrice": 123,
    "quantity": 1,
    "totalPrice": 123,
    "calculatedTaxes": [
      {
        "tax": 8.05,
        "taxRate": 7,
        "price": 123,
        "extensions": []
      }
    ],
    "taxRules": [
      {
        "taxRate": 7,
        "percentage": 100,
        "extensions": []
      }
    ],
    "referencePrice": null,
    "listPrice": null,
    "extensions": []
  },
  "price": [ // Disregard those, they are irrelevants"
  ],
  "listingPrice": [ // Disregard those, they are irrelevant
  ]
}
```

**Comment by patzick** [comment]
Will be implemented with #665
